moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -531,59 +270,17 @@ const ProjectsListViewTable = () => {
         {/*Main Table Body*/}
         <Paper className={classes.paper}>
           <Box mt={3}>
-            {loading ? (
+            {!data && !error ? (
```

Comment:
because we are now using a `cache-and-network` fetch policy, we must stop listening on `loading` - since we want the loading to happen in the background so long as we any data to render in the list.

Reply:
- https://github.com/cityofaustin/atd-data-tech/issues/14310

Reply:
not waiting on loading seems like a great improvement 😍